floss/main.py

```diff
@@ -356,13 +356,20 @@ def is_supported_file_type(sample_file_path: Path):
     :return: True if file type is supported, False otherwise
     """
     with sample_file_path.open("rb") as f:
-        magic = f.read(2)
+        magic = f.read(4)
 
     if magic in SUPPORTED_FILE_MAGIC:
         return True
+    elif magic[:2] in SUPPORTED_FILE_MAGIC:
+        return True
     else:
         return False
 
+def get_file_type(sample_file_path: Path):
```

Comment:
this seems to duplicate some of the logic from the function above, can you combine the logic or rewrite this?